JSDemos/Demos/Gauges/Overview/jQuery/description.md

```diff
@@ -1 +1,3 @@
-DevExtreme JavaScript Gauge controls are interactive jQuery components for visualizing data on dashboards. They come in two geometric shapes: circular, or radial, and linear. Circular gauges, in turn, can be of two sub-types: regular gauges and bar gauges. In this demo, you can see a simple dashboard with several highly customized gauges. Check out other demos in this section to explore gauge features, like geometry customization, palettes, tooltips, etc.
\ No newline at end of file
+DevExtreme JavaScript Gauge controls are interactive jQuery components for visualizing data on dashboards. They come in two geometric shapes: circular, or radial, and linear. Circular gauges, in turn, can be of two sub-types: regular gauges and bar gauges. In this demo, you can see a simple dashboard with several highly customized gauges. 
+
+Check out other demos in this section to explore gauge features, like geometry customization, palettes, tooltips, custom labels in the component's center, etc.
```

Comment:
I wanted to introduce a consistent template for all platform description. Right now, some descriptions are similar to others, some are quite different, although they all seem to say the same things. So I added a consistent basic template and then describe platform-specific functionality in the end. 

Since jQuery is the simplest case, this is the basic template that I use in all suggestions: 

```suggestion
DevExtreme JavaScript Gauge controls help you visualize data and create dashboards. Our library includes three gauge types: radial bar gauges, circular scale gauges, and linear gauges. 

This demo displays a simple dashboard with a few highly customized gauges. 

Review other demos in this section to explore gauge features such as geometry customization, palettes, tooltips, and custom label display.
```